xbmc/pvr/PVRManager.cpp

```diff
@@ -46,10 +46,22 @@ using namespace PVR;
 using namespace KODI::MESSAGING;
 using namespace std::chrono_literals;
 
-namespace PVR
+namespace
+{
+
+class CPVRJob
 {
+public:
+  virtual ~CPVRJob() = default;
+
+  virtual bool DoWork() = 0;
+  virtual const std::string GetType() const = 0;
```

Comment:
```suggestion
  virtual const std::string& GetType() const = 0;
```

Reply:
No, this way you force any potential importation if that method to store the value to be returned here in a merger variable it even worse a global static variable

Reply:
makes sense, lesson learned. thx